notebooks/MosvizNIRISSExample.ipynb

```diff
@@ -77,19 +77,14 @@
   },
   {
    "cell_type": "raw",
-   "id": "a6324128",
-   "metadata": {
-    "ExecuteTime": {
-     "end_time": "2022-09-27T19:07:34.004359Z",
-     "start_time": "2022-09-27T19:07:34.001488Z"
-    }
-   },
+   "id": "cafb85d1",
+   "metadata": {},
    "source": [
     "# *** OR ***\n",
     "# Change this cell type to Code and run this cell if you have a different data path.\n",
     "# Replace the value with your real path.\n",
     "\n",
-    "data_dir = '/Users/username/Data/NIRISS/'"
+    "data_dir = '/Users/rosteen/Data/NIRISS/'"
```

Comment:
not sure this was intentional?  Maybe we can rely on a shortcut to the user directory (either `os.path.expanduser` or `~/`) so that this can at least run locally for you without modifications?  Or can we just use a temporary directory instead to avoid hardcoding a path at all?

Reply:
Not intentional, good catch. Removed a couple other lines from testing that I accidentally committed again.